scripts/validation/providers/schema.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
@bastienbeurier this is just a start, but we should expand on this as we go along